packages/v4/patternfly-docs/content/design-guidelines/components/select/select.md

```diff
@@ -61,7 +61,7 @@ Both the checkbox select and multiple select lists allow users to select multipl
 
 <img src="./img/suggested-multiselect.png" alt="Example for using a multiple select inside a form" width="518"/>
 
-## Select list features
+## Variations
```

Comment:
Would make re-word the first lines of each sub usage section to say "Use Single select when ...." and then continue with the description so that it's more "Usage" based if that makes sense